### DISCUSSION TITLE
Categories header for listing

### DISCUSSION THREAD

**Discussion by robjhyndman** [quarto-dev/quarto-cli#2722] [Q&A] [answered]
Is it possible to change the header "Categories" that appears in the side bar for a listing when `categories: true` is set in the yaml? For example, can I set it to "Topics"?

**Comment by dragonstyle** [comment, marked as answer]
You should be able to place the following in your front matter:

```markdown
language: 
  listing-page-field-categories: "Topics"
```

This provides the string used in the heading.
